actions/user_actions.test.js

```diff
@@ -5,10 +5,14 @@ import thunk from 'redux-thunk';
 import configureStore from 'redux-mock-store';
 
 import {Preferences} from 'mattermost-redux/constants';
+import channelCategories from 'mattermost-redux/selectors/entities/channel_categories';
+import {CategoryTypes} from 'mattermost-redux/constants/channel_categories';
 
 import * as UserActions from 'actions/user_actions';
 
 const mockStore = configureStore([thunk]);
+const mockChannelsObj1 = [{id: 'gmChannel', type: 'G1'}];
```

Comment:
If these are supposed to be gm channels, shouldn't those types be using `General.GM_CHANNEL`? `G1` and `G2` aren't valid channel types.

Reply:
It was a mock so it wouldn't have mattered all that much. That wasn't great in the way I had to hack in the first place. I spent quite a lot of time trying to figure out how to pass out of scope variable to mock and there is no way to do that.

Reply:
I'm confused. We're mocking the constants? This test file is a mess, but I didn't know it was that bad.